modules/openapi-generator-online/src/main/java/org/openapitools/codegen/online/service/Generator.java

```diff
@@ -118,6 +118,12 @@ private static String generate(String language, GeneratorInput opts, Type type)
             throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "The OpenAPI specification supplied was not valid");
         }
 
+//        if(opts.getOpenapiNormalizer() != null && !opts.getOpenapiNormalizer().isEmpty()){
+//            for(String ruleString: opts.getOpenapiNormalizer()) {
+//
+//            }
+//        }
+
```

Comment:
thanks for the PR

do we still need these commented code?

Reply:
ok, removed it and also formatted some pieces of code.